### ISSUE TITLE
Renaming model breaks all model chains

### ISSUE THREAD

**Issue by timothylarson** [smeighan/xLights#1664]
I just recorded a video showing the problem, basically when I rename any prop, all of my model chains break.

Here is the video https://youtu.be/kqfV7cQdkmg

I am running xLights Ver 2019.41 on a Mac.

And here are my show backup files before I have the error:
[Keybindings and Network.zip](https://github.com/smeighan/xLights/files/3517635/Keybindings.and.Network.zip)

**Comment by keithsw1111** [comment]
i think i have this fixed for .42

**Comment by timothylarson** [comment]
I look forward to the update. 

Thanks so much Keith!

**Comment by keithsw1111** [comment]
Please reopen if ,42 did not fix it

**Comment by timothylarson** [comment]
Hey Keith, I just downloaded and tested version .42 but I am still having the same issue. 

Creating new groups and models is fine, it's just when I rename things that everything gets messed up.

I tested this on a PC and am still having the same issue.

I've been able to rename props using a text editor and using "search and replace" within the xml file, but I can't change the names in xLights itself without my model chains getting messed up.

Here's an updated version of my layout.

[xLights Layout - 2019-08-21.zip](https://github.com/smeighan/xLights/files/3527856/xLights.Layout.-.2019-08-21.zip)
